manifests/init.pp

```diff
@@ -125,5 +125,4 @@
   Class['certs::ca']
 
   $default_ca = $certs::ca::default_ca
-  $server_ca = $certs::ca::server_ca
```

Comment:
Note this is used here:
https://github.com/theforeman/puppet-foreman_proxy_content/blob/37a6e9eaa058fefd1788625ad5def7243cb351e5/manifests/bootstrap_rpm.pp#L15